custom_components/jcihitachi_tw/__init__.py

```diff
@@ -24,6 +24,10 @@
 BASE_TIMEOUT = 5
 
 
+class JciHitachiDataUpdateCoordinator(DataUpdateCoordinator):
+    def set_updated_data(self, device=None) -> None:
+        self.hass.loop.call_soon_threadsafe(self.async_set_updated_data, device)
```

Comment:
Hmm, `device` looks to me unneeded no?

Reply:
Yes, for this implementation we can default to None and it should be ok. The `device` is there to be consistent with the original `async_set_updated_data` function. I can remove it if you prefer it that way.

Reply:
See below, thanks.